libs/java/server_common/src/main/java/com/yahoo/athenz/common/server/notification/NotificationToEmailConverterCommon.java

```diff
@@ -111,6 +111,18 @@ public String generateBodyFromTemplate(Map<String, String> metaDetails,
         return addCssStyleToBody(bodyString);
     }
 
+    public String getTableEntryTemplate(int numOfColumns) {
```

Comment:
This is just a small improvement I had to do after I entered this template as a resource for the third time :)